eng/common/TestResources/New-TestResources.ps1

```diff
@@ -76,7 +76,10 @@ param (
     [switch] $Force,
 
     [Parameter()]
-    [switch] $OutFile
+    [switch] $OutFile,
+
+    [Parameter()]
+    [boolean] $DevopsLogging = ($null -ne $env:SYSTEM_TEAMPROJECTID)
```

Comment:
Don't use `[boolean]` parameters in PowerShell. Recommendation is `[switch]`. They can be turned off universally:

* `-MySwitch` is equivalent to `-MySwitch:$true`
* To disable, pass `-MySwitch:$false`

This script is meant to be run manually as well and, as such, meant to provide a consistent interface like all other cmdlets people might type in a shell.

Reply:
Ah, I forgot about the `-switch:$bool` syntax.

Reply:
And `SwitchParameter`s are coercible to booleans, so you just use them like booleans e.g. `if ($MySwitch)` or `if (!$MySwitch)`, etc.